launcher

```diff
@@ -401,6 +401,9 @@ run_start(){
    ports=`cat $config_file | $docker_path run $user_args --rm -i -a stdout -a stdin $image ruby -e \
           "require 'yaml'; puts YAML.load(STDIN.readlines.join)['expose'].map{|p| '-p ' << p.to_s << ' '}.join"`
 
+   docker_args=`cat $config_file | $docker_path run $user_args --rm -i -a stdout -a stdin $image ruby -e \
+          "require 'yaml'; puts YAML.load(STDIN.readlines.join)['docker_args']"`
```

Comment:
TL;DR. Yes, but it works just like in `ports`, so no problem.

In fact, during command substitution (processing backticks) bash chomps [any trailing newlines](https://www.gnu.org/software/bash/manual/html_node/Command-Substitution.html#Command-Substitution), so the newline doesn't even appear in `$docker_args`.

Removal of trailing newlines explains the following output:
```bash
$ echo foo
foo
$ echo $(echo foo)
foo
$ ruby -e 'puts "foo"'
foo
$ echo $(ruby -e 'puts "foo"')
foo
$ ruby -e 'puts "foo"; puts ""'
foo

$ echo $(ruby -e 'puts "foo"; puts ""')
foo
```